src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -168,10 +162,16 @@ export class AdditionalServicesUtils {
           atIndex = idx;
         }
       } else {
+        let scheduleStatus;
+        if (foundSchedule) {
+          scheduleStatus = foundSchedule?.enabled === true ? 'running' : 'disabled';
```

Comment:
this is not  scheduleStatus but mix of service status and schedule state

Reply:
This is the scheduleStatus, which we will show when service is not in failed Or unresponsive state.